f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -835,10 +836,12 @@ def annotate_service_members(self, service):
                 self.network_builder._annotate_service_route_domains(service)
             except f5ex.InvalidNetworkType as exc:
                 LOG.warning(exc.message)
+            except HTTPError as err:
```

Comment:
Does this change improve anything?

Reply:
It will throw the original HTTPError, rather cover up by f5ex.RouteDomainCreationException.
The caller will not tell what Exception happens exactly.